venia-integration-tests/src/actions/productPage/index.js

```diff
@@ -20,3 +21,17 @@ export const addProductToExistingWishlistFromDialog = wishlistName => {
         .contains(wishlistName)
         .click();
 };
+
+/**
+ * Utility function to add simple product to cart from the product page
+ *
+ * Note: This only works for simple product because it doesnt consider
+ * product options. A different funciton has to be used for different
```

Comment:
```suggestion
 * product options. A different function has to be used for different
```

I'm sorry I can't help it I was born this way 😭

Reply:
Dude, thank you 🤣 I am laughing at myself.